loader/windows/icd_windows.c

```diff
@@ -17,14 +17,67 @@
  */
 
 #include "icd.h"
+#include <windows.h>
```

Comment:
Requiring a specific order for #include files is almost always an indication that something is wrong.  If icd_windows_hkr.h or icd_windows_dxgk.h requires windows.h then they should include it themselves, vs. requiring windows.h to be included before them.

Reply:
Added windows.h in icd_windows_hkr.h.

Reply:
Great, we change the include file order back here then.